ftw/recipe/deployment/logrotate.py

```diff
@@ -79,6 +79,43 @@ def create_logrotate_conf(recipe):
 
         logrotate_conf.write('}\n')
 
+    # Add configuration for ftw.jsonlog logs
+    for zope_part in recipe.zope_parts:
+        logs = []
+        event_log = '%s/var/log/%s.log' % (recipe.buildout_dir, zope_part)
+        event_log = recipe.buildout[zope_part].get('event-log', event_log)
+        # ftw.jsonlog's log path is based on eventlog path
+        jsonlog_log = event_log.replace('.log', '-json.log')
+        if event_log:
+            logs.append(jsonlog_log)
```

Comment:
doesn't make sense to have a list of logs here as there will be only one jsonlog.